tuna/miopen/fin_builder.py

```diff
@@ -132,6 +152,10 @@ def step(self):
 
     if failed_job:
       self.set_job_state('errored', result=result_str)
+    elif self.pending:
+      self.set_job_state('compiled_pend', result=result_str)
```

Comment:
I am not familiar with this state, can you elaborate what this represents ?

Reply:
Pending is a list containing results that need to be inserted into the database. If it has members for the current job at the end of the step then they are added to the result queue, and the job is noted as compiled_pend. When the result queue is drained, the state will be updated to compiled.